### ISSUE TITLE
Uitschrijven leden stappen automatiseren

### ISSUE THREAD

**Issue by indiePeeters** [esac-ic/esac.nl#68]
Het uitschrijven van leden gaat als volgt:

1. Uitschrijven in online ledensysteem (ESAC Website)
2. Verwijderen van SSC ledenlijst
3. Bijwerken inschrijvingen en uitschrijvingen sheet (een sheet met naam datuminschrijving, datumuitschrijving, lidmaatschap, en twee checks voor verwijderen boekhouding en ssc ledenlijst)
4. Wanneer van toepassing formulier verwijderen uit smoelenboek
5. Bevestigingsmail sturen naar lid

Het momenteel een beetje stom dat deze zaken handmatig moeten worden uitgevoerd worden als vrijwel alle bovenstaande zaken geautomatiseerd kunnen worden. 

Ik zie nu zo het stappenplan voor me
1. Uitschrijven in online ledensysteem (ESAC Website) (Met een check voor SSC Ledenlijst.)
2. Data automatisch verwerkt in uitschrijvingen tabel. (een sheet ergens online zal ook voldoen)
3. Automatische bevestigingsmail sturen naar lid.

hierdoor zullen enkel de volgende zaken handmatig blijven:
- Verwijderen uit SSC ledenlijst.
- Eventueel verwijderen uit smoelenboek.

De laatste stap is voor de penningmeester, het lid moet dan nog uit de boekhouding verwerkt worden. Dit kan opgelost worden om een mail naar de penningmeester te sturen dat dit gedaan moet worden

**Comment by otherwise777** [comment]
wat houd put 3 van de uitschrijving in? Is dit een bestand wat ergens buiten de website staat? de inschrijvingen en uitschrijvingen sheet

**Comment by indiePeeters** [comment]
Yep, het is een excel bestand wat door de secretaris bijgehouden wordt, dit dient meer als een check zodat de uitschrijving goed verloopt.

Dit zijn de kolommen in de excel sheet.

Naam | Lidsoort | Lid vanaf | Lid tot | Verwijderd van maillijsten (ja/nee) | Verwijderd van SSC ledenlijst (ja/nee) | Verwijderd uit boekhouding | Uitgevoerd op
